src/main.rs

```diff
@@ -17,8 +19,20 @@ use probe_rs::{
     DebugProbeError, DebugProbeSelector, Probe, WireProtocol,
 };
 
+#[cfg(feature = "sentry")]
+use probe_rs_cli_util::logging::{ask_to_log_crash, capture_anyhow, capture_panic, Metadata};
 use probe_rs_cli_util::{argument_handling, build_artifact, logging, read_metadata};
 
+#[cfg(feature = "sentry")]
```

Comment:
Would it make sense to keep this always enabled? Even if sentry is not used, it might be useful to add the git version and the release number to the log, or the version output for example.

Reply:
Yeah I guess we can do that :)